api/v1alpha1/modelregistry_types.go

```diff
@@ -135,10 +188,15 @@ type ModelRegistrySpec struct {
 	// Configuration for gRPC endpoint
 	Grpc GrpcSpec `json:"grpc"`
 
-	//+kubebuilder:required
+	// One of Postgres or MySQL database connections must be specified!
```

Comment:
is there a way we can maybe add this as part of RestSpec comment, so to see it in resulting docs?

in other words my question is: if this comment is left here, will it be present in resulting generated docs, or maybe best to place this comment on top of line `type RestSpec struct {` 137 so to have it included?